implementations/micrometer-registry-influx/src/main/java/io/micrometer/influx/InfluxMeterRegistry.java

```diff
@@ -74,6 +77,12 @@ private void createDatabaseIfNecessary() {
             con.setConnectTimeout((int) config.connectTimeout().toMillis());
             con.setReadTimeout((int) config.readTimeout().toMillis());
             con.setRequestMethod(HttpMethod.POST);
+
+            if (!config.sslVerify() && "https".equals(queryEndpoint.getProtocol())) {
+                HttpsURLConnection.class.cast(con).setSSLSocketFactory(createSslSocketFactory());
+                HttpsURLConnection.class.cast(con).setHostnameVerifier(new NoopHostnameVerifier());
+            }
```

Comment:
It would be nice to extract a method

    private void doNotVerifySslCeritifcateIfDesired(HttpURLConnection con, URL url) {
      if (!config.sslVerify() && "https".equals(url.getProtocol())) {
        HttpsURLConnection httpsCon = HttpsURLConnection.class.cast(con);
        httpsCon.setSSLSocketFactory(createSslSocketFactory());
        httpsCon.setHostnameVerifier(new NoopHostnameVerifier());
      }
    }

because it makes it easier to understand the code and it avoids duplicate code.

Reply:
Yes, it does make sense. Thanks!